### ISSUE TITLE
Possibility of F2FS Root?

### ISSUE THREAD

**Issue by owenthewizard** [ccrisan/thingos#2]
Is there any reason F2FS (could|should)n't be used for the root file system?

**Comment by ccrisan** [comment]
Nope, it's just due to my lack of any experience with it :) In theory, with minimal changes it should work.
